editor/editor_log.cpp

```diff
@@ -419,6 +428,11 @@ EditorLog::EditorLog() {
 	vb_right->add_child(std_filter->toggle_button);
 	type_filter_map.insert(MSG_TYPE_STD, std_filter);
 
+	LogFilter *rich_filter = memnew(LogFilter(MSG_TYPE_STD_RICH));
+	rich_filter->initialize_button(TTR("Toggle visibility of rich output messages."), callable_mp(this, &EditorLog::_set_filter_active));
+	vb_right->add_child(rich_filter->toggle_button);
+	type_filter_map.insert(MSG_TYPE_STD_RICH, rich_filter);
```

Comment:
Should rich logs be considered a different type of logging for the purpose of filtering? I'm not sure if users expect it to be considered a different message category.

Instead, I'd make the standard log button toggle the display of both non-rich and rich prints.

Reply:
Okay, I work on it as soon as I can! :)

Reply:
Fixed it and working perfectly like before.